v3/fleet-local/it-hybrid/memcached.service

```diff
@@ -10,7 +10,7 @@ Restart=always
 RestartSec=5
 TimeoutStartSec=0
 
-ExecStartPre=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && docker pull borja/docker-memcached"
+ExecStartPre=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && docker pull index.docker.io/borja/docker-memcached"
```

Comment:
@bryanlatten Technically, yes - but based on discussions we've had so far, we will always use "latest" in mesos-systemd. If a lock is needed (up to the launching user's discretion), it can be defined in the configs section of the secrets.json file.